src/App.jsx

```diff
@@ -1,31 +1,56 @@
-
-import React from 'react';
-import Overview from './components/overview/Overview.jsx';
-import Reviews from './components/reviews/Reviews.jsx';
-import Questions from './components/questions/Questions.jsx';
-import Related from './components/related/Related.jsx';
+import React from "react";
+import axios from "axios";
+import Overview from "./components/overview/Overview.jsx";
+import Reviews from "./components/reviews/Reviews.jsx";
+import Questions from "./components/questions/Questions.jsx";
+import Related from "./components/related/Related.jsx";
+import API_KEY from "./config.js";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      productId: 11 // TODO - find out what this should actually be.
+      productId: null
     };
     this.updatedProductId = this.updatedProductId.bind(this);
```

Comment:
Thanks for the name change, Bryan. Looks like the binding name still needs to be updated though